src/passes/MinifyImportsAndExports.cpp

```diff
@@ -172,6 +168,12 @@ struct MinifyImportsAndExports : public Pass {
         process(curr->base);
       }
     };
+    if (module->memory.exists) {
+      processImport(&module->memory);
+    }
+    if (module->table.exists) {
+      processImport(&module->table);
+    }
```

Comment:
Could we just use `iterImportedMemories` and `iterImportedTables`?

Or better still add an `iterImports` and just use that?

Reply:
Ah, good idea for `iterImports`, that's much nicer. Added now.